critiquebrainz/frontend/views/__init__.py

```diff
@@ -0,0 +1,11 @@
+import critiquebrainz.db.avg_rating as db_avg_rating
+import critiquebrainz.db.exceptions as db_exceptions
+
+def get_avg_rating(entity_id, entity_type):
+    """Retrieve avg_rating and convert rating on a scale 1-5."""
```

Comment:
What's `avg_rating`? Average rating? There's no need to literally use variable names in docstrings.